### PR TITLE
Fix send invoice logic and linked unit tests

### PR DESCRIPTION
**Pull request by dexon44**
I want to merge this change because it fix send invoice logic.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3993 Missing channel_slug in send_invoice](https://app.clickup.com/t/2549495/SALEOR-3993)

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: saleor/invoice/notifications.py [hunk 1]
```diff
@@ -33,5 +33,11 @@ def send_invoice(
         "requester_app_id": app.id if app else None,
         **get_site_context(),
     }
-    manager.notify(NotifyEventType.INVOICE_READY, payload)  # type: ignore
+
+    channel_slug = None
```
Comment (fowczarek) [inline saleor/invoice/notifications.py:37]: For invoices with removed orders we still sending multiple notifications. I'm not sure it's proper behavior.

**Review by fowczarek — COMMENTED** [review]
You also should remove `requirements_dev.txt` and `requirements.txt` from this PR :police_car:

**Comment by maarcingebala** [comment]
@dexon44 This PR should be rebased against the newest 3.0 branch. Now it contains commits that don't belong to this PR.

**Pull request closed without merging**